src/GeometricMean/GeometricMean.sol

```diff
@@ -92,6 +92,10 @@ contract GeometricMean is PairStrategy {
             data, (uint256, uint256, uint256, uint256, uint256, address)
         );
 
+        if (pool.reserves.length != 2) {
```

Comment:
Why is this one different from the others?

Reply:
oh yeah is it missing the reserves.length check?

Reply:
This one is different because it's expecting reserveX and reserveY as individual variables instead of an array (like in G3M or LogNormal).

If you guys are down I suggest that we use an array for this strategy too to keep things consistent.

Reply:
Im down

Reply:
> Im down

It's done!